cloud-dataplane/x-topics/quickstart.md

```
@@ -1,6 +1,6 @@
-The following steps describe how to authenticate with the Data Plane APIs and create a new topic. For more information on the Data Plane APIs, see the Cloud API Overview.
+The following steps describe how to authenticate with the Data Plane APIs and create a new topic. For more information on the Data Plane APIs, see the [Cloud API Overview](#topic-cloud-api-overview).
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid

(MD051, link-fragments)